packages/sdk-core/package.json

```diff
@@ -57,7 +57,7 @@
     },
     "dependencies": {
         "@superfluid-finance/ethereum-contracts": "1.9.0",
```

Comment:
This is also locked down, and interestingly 1.9.0 is not released yet

Reply:
We have talked about this. This is the currently how we test sdk-core compatibility with the unreleased ethereum-contracts.

Happy to open this topic in the future again, but not changing workflow of this for now.